deploy/kubernetes/charts/serai/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: serai
+description: A Helm chart for serai
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```

Comment:
AFAICT, this is the Serai version. Serai isn't versioned as 1.16.0. Mind clarifying what the field actually is, if different, or correcting the Serai version?

Reply:
deleted appVersion tag from yaml files.